### PR TITLE
fix: Fixed typo in the path

### PR DESCRIPTION
**Pull request by andi4191**
Signed-off-by: Anurag Dixit <a.dixit91@gmail.com>

# Description
This typo is causing issue in CI pipeline. Fixed it

Fixes # (issue)

## Type of change

Please delete options that are not relevant and/or add your own.

- Bug fix (non-breaking change which fixes an issue)
- 
# Checklist:

- [x] My code follows the style guidelines of this project (You can use the linters)
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas and hacks
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests to verify my fix or my feature
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by narendasan** [comment]
We need both just in case people havent recloned the repo. Its just a search dir. why is it breaking CI?

**Comment by andi4191** [comment]
> We need both just in case people havent recloned the repo. Its just a search dir. why is it breaking CI?

A user doesn't need to reclone the repo. They can simply rename their fork. Github has an option to update the name of the fork.
Without this change the CI pipeline is failing to locate the TensorRT header file "NvInfer.h". With this change it is running fine.

**Comment by narendasan** [comment]
No the issue is that bazel uses the name of the parent directory (Torch-TensorRT on a fresh clone) for the name of the workspace. If you have an existing clone it will still call the directory bazel-trtorch even though the repo has a different name

**Comment by narendasan** [comment]
image

File: py/setup.py [hunk 1]
```diff
@@ -239,7 +239,7 @@ def run(self):
         libraries=["torchtrt"],
         include_dirs=[
             dir_path + "torch_tensorrt/csrc", dir_path + "torch_tensorrt/include",
-            dir_path + "/../bazel-TRTorch/external/tensorrt/include",
+            dir_path + "/../bazel-Torch-TensorRT/external/tensorrt/include",
```
Comment (peri044) [inline py/setup.py:242]: we already have this in the line 243. Last time I checked with @narendasan , we wanted to keep TRTorch here to support users who didn't clone the repo (after the rebranding). Those users will still have the directory name TRTorch and corresponding bazel dirs names.

**Comment by andi4191** [comment]
Closing this PR for backward compatibility.

**Pull request closed without merging**